--- origsrc/evolution-2.26.2/a11y/Makefile.am	2009-05-03 09:59:54.000000000 -0500
+++ src/evolution-2.26.2/a11y/Makefile.am	2009-06-16 19:29:43.789712400 -0500
@@ -1,7 +1,7 @@
 # Somewhat odd looking to have "." in SUBDIRS, but apparently it works?
 SUBDIRS = e-text e-table . calendar widgets addressbook
 
-if OS_WIN32
+if PLATFORM_WIN32
 WIN32_BOOTSTRAP_LIBS =					\
 	$(top_builddir)/win32/libemiscwidgets.la	\
 	$(top_builddir)/win32/libetable.la		\
--- origsrc/evolution-2.26.2/a11y/addressbook/Makefile.am	2009-05-03 09:59:54.000000000 -0500
+++ src/evolution-2.26.2/a11y/addressbook/Makefile.am	2009-06-16 19:29:43.792212500 -0500
@@ -1,4 +1,4 @@
-if OS_WIN32
+if PLATFORM_WIN32
 WIN32_BOOTSTRAP_LIBS =						\
 	$(top_builddir)/win32/libemiscwidgets.la		\
 	$(top_builddir)/win32/libevolution-addressbook.la
--- origsrc/evolution-2.26.2/a11y/calendar/Makefile.am	2009-05-03 09:59:54.000000000 -0500
+++ src/evolution-2.26.2/a11y/calendar/Makefile.am	2009-06-16 19:29:43.794712500 -0500
@@ -3,7 +3,7 @@
 # for debug
 #A11Y_CFLAGS += -pedantic -ansi -DACC_DEBUG -Werror
 
-if OS_WIN32
+if PLATFORM_WIN32
 WIN32_BOOTSTRAP_LIBS = 					\
 	$(top_builddir)/win32/libetext.la		\
 	$(top_builddir)/win32/libevolution-calendar.la
--- origsrc/evolution-2.26.2/a11y/widgets/Makefile.am	2009-05-03 09:59:54.000000000 -0500
+++ src/evolution-2.26.2/a11y/widgets/Makefile.am	2009-06-16 19:29:43.797212600 -0500
@@ -2,7 +2,7 @@
 # for debug
 #A11Y_CFLAGS += -pedantic -ansi -DACC_DEBUG -Werror
 
-if OS_WIN32
+if PLATFORM_WIN32
 WIN32_BOOTSTRAP_LIBS = $(top_builddir)/win32/libemiscwidgets.la
 endif
 
--- origsrc/evolution-2.26.2/addressbook/gui/component/Makefile.am	2009-05-03 09:59:54.000000000 -0500
+++ src/evolution-2.26.2/addressbook/gui/component/Makefile.am	2009-06-16 19:29:43.802212600 -0500
@@ -1,4 +1,4 @@
-if OS_WIN32
+if PLATFORM_WIN32
 WIN32_BOOTSTRAP_LIBS = $(top_builddir)/win32/libevolution-mail.la
 endif
 
--- origsrc/evolution-2.26.2/addressbook/gui/contact-editor/Makefile.am	2009-05-03 09:59:54.000000000 -0500
+++ src/evolution-2.26.2/addressbook/gui/contact-editor/Makefile.am	2009-06-16 19:29:43.803462700 -0500
@@ -1,4 +1,4 @@
-if OS_WIN32
+if PLATFORM_WIN32
 WIN32_BOOTSTRAP_LIBS = $(top_builddir)/win32/libevolution-addressbook.la
 endif
 
--- origsrc/evolution-2.26.2/addressbook/gui/contact-list-editor/Makefile.am	2009-05-03 09:59:54.000000000 -0500
+++ src/evolution-2.26.2/addressbook/gui/contact-list-editor/Makefile.am	2009-06-16 19:29:43.805962700 -0500
@@ -1,4 +1,4 @@
-if OS_WIN32
+if PLATFORM_WIN32
 WIN32_BOOTSTRAP_LIBS = $(top_builddir)/win32/libevolution-addressbook.la
 endif
 
--- origsrc/evolution-2.26.2/calendar/gui/Makefile.am	2009-05-03 09:59:55.000000000 -0500
+++ src/evolution-2.26.2/calendar/gui/Makefile.am	2009-06-16 19:29:43.810962800 -0500
@@ -1,4 +1,4 @@
-if OS_WIN32
+if PLATFORM_WIN32
 WIN32_BOOTSTRAP_LIBS = $(top_builddir)/win32/libevolution-mail.la
 endif
 
--- origsrc/evolution-2.26.2/e-util/Makefile.am	2009-05-03 09:59:55.000000000 -0500
+++ src/evolution-2.26.2/e-util/Makefile.am	2009-06-16 19:29:43.819712900 -0500
@@ -3,8 +3,10 @@ econdincludedir = $(privincludedir)/e-co
 ecpsdir = $(privdatadir)/ecps
 ruledir = $(privdatadir)
 
-if OS_WIN32
+if PLATFORM_WIN32
 WIN32_BOOTSTRAP_LIBS = $(top_builddir)/win32/libemiscwidgets.la
+endif
+if OS_WIN32
 PLATFORM_SOURCES = e-win32-reloc.c
 endif
 
--- origsrc/evolution-2.26.2/mail/importers/Makefile.am	2009-05-03 09:59:55.000000000 -0500
+++ src/evolution-2.26.2/mail/importers/Makefile.am	2009-06-16 19:29:43.823463000 -0500
@@ -1,4 +1,4 @@
-if OS_WIN32
+if PLATFORM_WIN32
 WIN32_BOOTSTRAP_LIBS = $(top_builddir)/win32/libevolution-mail.la
 endif
 
--- origsrc/evolution-2.26.2/plugins/audio-inline/Makefile.am	2009-05-03 09:59:55.000000000 -0500
+++ src/evolution-2.26.2/plugins/audio-inline/Makefile.am	2009-06-16 19:29:43.832213100 -0500
@@ -1,3 +1,9 @@
+if PLATFORM_WIN32
+NO_UNDEFINED_REQUIRED_LIBS =	\
+	$(top_builddir)/mail/libevolution-mail.la	\
+	$(top_builddir)/e-util/libeutil.la
+endif
+
 INCLUDES =						\
 	-I$(top_srcdir)					\
 	$(EVOLUTION_MAIL_CFLAGS)			\
@@ -11,6 +17,7 @@ plugin_LTLIBRARIES = liborg-gnome-audio-
 liborg_gnome_audio_inline_la_SOURCES = audio-inline.c
 liborg_gnome_audio_inline_la_LDFLAGS = -module -avoid-version $(NO_UNDEFINED)
 liborg_gnome_audio_inline_la_LIBADD =	\
+	$(NO_UNDEFINED_REQUIRED_LIBS)	\
 	$(GSTREAMER_LIBS)		\
 	$(EVOLUTION_MAIL_LIBS)
 
--- origsrc/evolution-2.26.2/plugins/bogo-junk-plugin/Makefile.am	2009-05-03 09:59:55.000000000 -0500
+++ src/evolution-2.26.2/plugins/bogo-junk-plugin/Makefile.am	2009-06-16 19:29:43.842213200 -0500
@@ -1,3 +1,8 @@
+if PLATFORM_WIN32
+NO_UNDEFINED_REQUIRED_LIBS =			\
+	$(top_builddir)/mail/libevolution-mail.la
+endif
+
 INCLUDES =						\
 	-I$(top_srcdir)					\
 	-DWELCOME_MESSAGE=\""$(privdatadir)/default/C/mail/local/Inbox"\" 	\
@@ -11,6 +16,7 @@ plugin_LTLIBRARIES = liborg-gnome-bogo-j
 liborg_gnome_bogo_junk_plugin_la_SOURCES = bf-junk-filter.c
 liborg_gnome_bogo_junk_plugin_la_LDFLAGS = -module -avoid-version $(NO_UNDEFINED)
 liborg_gnome_bogo_junk_plugin_la_LIBADD =	\
+	$(NO_UNDEFINED_REQUIRED_LIBS)	\
 	$(GNOME_PLATFORM_LIBS) \
 	$(top_builddir)/e-util/libeutil.la
 
--- origsrc/evolution-2.26.2/plugins/external-editor/Makefile.am	2009-05-03 09:59:55.000000000 -0500
+++ src/evolution-2.26.2/plugins/external-editor/Makefile.am	2009-06-16 19:29:43.869713700 -0500
@@ -1,4 +1,4 @@
-if OS_WIN32
+if PLATFORM_WIN32
 NO_UNDEFINED_REQUIRED_LIBS = \
 	$(EVOLUTION_MAIL_LIBS) \
 	$(GNOME_PLATFORM_LIBS) \
--- origsrc/evolution-2.26.2/plugins/groupwise-features/Makefile.am	2009-05-03 09:59:55.000000000 -0500
+++ src/evolution-2.26.2/plugins/groupwise-features/Makefile.am	2009-06-16 19:29:43.883463900 -0500
@@ -1,4 +1,4 @@
-if OS_WIN32
+if PLATFORM_WIN32
 NO_UNDEFINED_REQUIRED_LIBS = $(top_builddir)/mail/libevolution-mail.la
 endif
 
--- origsrc/evolution-2.26.2/plugins/imap-features/Makefile.am	2009-05-03 09:59:55.000000000 -0500
+++ src/evolution-2.26.2/plugins/imap-features/Makefile.am	2009-06-16 19:29:43.890964000 -0500
@@ -1,4 +1,4 @@
-if OS_WIN32
+if PLATFORM_WIN32
 NO_UNDEFINED_REQUIRED_LIBS = $(top_builddir)/mail/libevolution-mail.la
 endif
 
--- origsrc/evolution-2.26.2/plugins/mail-notification/Makefile.am	2009-05-03 09:59:55.000000000 -0500
+++ src/evolution-2.26.2/plugins/mail-notification/Makefile.am	2009-06-16 19:29:43.902214200 -0500
@@ -1,4 +1,4 @@
-if OS_WIN32
+if PLATFORM_WIN32
 NO_UNDEFINED_REQUIRED_LIBS =				\
 	$(top_builddir)/mail/libevolution-mail.la	\
 	$(top_builddir)/e-util/libeutil.la		\
--- origsrc/evolution-2.26.2/plugins/pst-import/Makefile.am	2009-05-03 09:59:55.000000000 -0500
+++ src/evolution-2.26.2/plugins/pst-import/Makefile.am	2009-06-16 19:29:43.925964500 -0500
@@ -1,4 +1,4 @@
-if OS_WIN32
+if PLATFORM_WIN32
 NO_UNDEFINED_REQUIRED_LIBS =				\
 	$(top_builddir)/mail/libevolution-mail.la	\
 	$(top_builddir)/e-util/libeutil.la		\
--- origsrc/evolution-2.26.2/plugins/python/Makefile.am	2009-05-03 09:59:55.000000000 -0500
+++ src/evolution-2.26.2/plugins/python/Makefile.am	2009-06-17 11:30:49.834377400 -0500
@@ -9,8 +9,9 @@ plugin_DATA = org-gnome-evolution-python
 plugin_LTLIBRARIES = liborg-gnome-evolution-python.la
 
 liborg_gnome_evolution_python_la_SOURCES = python-plugin-loader.c python-plugin-loader.h
-liborg_gnome_evolution_python_la_LDFLAGS = -module -avoid-version
+liborg_gnome_evolution_python_la_LDFLAGS = -module -avoid-version $(NO_UNDEFINED)
 liborg_gnome_evolution_python_la_LIBADD =	\
+	$(top_builddir)/e-util/libeutil.la \
 	-lpthread -ldl -lutil -lm \
 	$(PY_LIBS) \
 	$(E_UTIL_LIBS)
--- origsrc/evolution-2.26.2/plugins/sa-junk-plugin/Makefile.am	2009-05-03 09:59:55.000000000 -0500
+++ src/evolution-2.26.2/plugins/sa-junk-plugin/Makefile.am	2009-06-16 19:29:43.937214700 -0500
@@ -1,3 +1,8 @@
+if PLATFORM_WIN32
+NO_UNDEFINED_REQUIRED_LIBS =			\
+	$(top_builddir)/mail/libevolution-mail.la
+endif
+
 INCLUDES =						\
 	-I$(top_srcdir)					\
 	$(EVOLUTION_MAIL_CFLAGS)
@@ -9,6 +14,10 @@ plugin_LTLIBRARIES = liborg-gnome-sa-jun
 
 liborg_gnome_sa_junk_plugin_la_SOURCES = em-junk-filter.c
 liborg_gnome_sa_junk_plugin_la_LDFLAGS = -module -avoid-version $(NO_UNDEFINED)
+liborg_gnome_sa_junk_plugin_la_LIBADD =	\
+	$(NO_UNDEFINED_REQUIRED_LIBS)	\
+	$(top_builddir)/e-util/libeutil.la	\
+	$(EVOLUTION_MAIL_LIBS)
 
 
 BUILT_SOURCES = $(plugin_DATA) $(error_DATA)
--- origsrc/evolution-2.26.2/plugins/save-attachments/Makefile.am	2009-05-03 09:59:55.000000000 -0500
+++ src/evolution-2.26.2/plugins/save-attachments/Makefile.am	2009-06-16 19:29:43.940964800 -0500
@@ -1,4 +1,4 @@
-if OS_WIN32
+if PLATFORM_WIN32
 NO_UNDEFINED_REQUIRED_LIBS =				\
 	$(top_builddir)/mail/libevolution-mail.la	\
 	$(EVOLUTION_MAIL_LIBS)				\
--- origsrc/evolution-2.26.2/plugins/tnef-attachments/Makefile.am	2009-05-03 09:59:55.000000000 -0500
+++ src/evolution-2.26.2/plugins/tnef-attachments/Makefile.am	2009-06-16 19:29:43.958465000 -0500
@@ -1,4 +1,4 @@
-if OS_WIN32
+if PLATFORM_WIN32
 NO_UNDEFINED_REQUIRED_LIBS = 				\
 	$(EVOLUTION_MAIL_LIBS)				\
 	$(GNOME_PLATFORM_LIBS)				\
--- origsrc/evolution-2.26.2/widgets/misc/Makefile.am	2009-05-03 09:59:55.000000000 -0500
+++ src/evolution-2.26.2/widgets/misc/Makefile.am	2009-06-16 19:29:43.969715200 -0500
@@ -1,4 +1,4 @@
-if OS_WIN32
+if PLATFORM_WIN32
 WIN32_BOOTSTRAP_LIBS = $(top_builddir)/win32/libfilter.la
 endif
 
--- origsrc/evolution-2.26.2/widgets/table/Makefile.am	2009-05-03 09:59:55.000000000 -0500
+++ src/evolution-2.26.2/widgets/table/Makefile.am	2009-06-16 19:29:43.972215200 -0500
@@ -1,4 +1,4 @@
-if OS_WIN32
+if PLATFORM_WIN32
 WIN32_BOOTSTRAP_LIBS =					\
 	$(top_builddir)/win32/libemiscwidgets.la	\
 	$(top_builddir)/win32/libetext.la
--- origsrc/evolution-2.26.2/widgets/text/Makefile.am	2009-05-03 09:59:55.000000000 -0500
+++ src/evolution-2.26.2/widgets/text/Makefile.am	2009-06-16 19:29:43.974715300 -0500
@@ -1,4 +1,4 @@
-if OS_WIN32
+if PLATFORM_WIN32
 WIN32_BOOTSTRAP_LIBS = $(top_builddir)/win32/libemiscwidgets.la
 endif
 
--- origsrc/evolution-2.26.2/win32/Makefile.am	2009-05-03 09:59:55.000000000 -0500
+++ src/evolution-2.26.2/win32/Makefile.am	2009-06-16 19:29:43.978465300 -0500
@@ -8,7 +8,7 @@ EXTRA_DIST =				\
 	libevolution-mail.def		\
 	libfilter.def
 
-if OS_WIN32
+if PLATFORM_WIN32
 BOOTSTRAP_LIBS =			\
 	libemiscwidgets.la		\
 	libetable.la			\
@@ -27,38 +27,38 @@ noinst_DATA = $(BOOTSTRAP_LIBS)
 # libtool's -module -avoid-version options don't get the -0 suffix.)
 
 libemiscwidgets.la: dummy.la libemiscwidgets.def
-	sed -e s!%DLL%!libemiscwidgets-0! -e s!%LIB%!libemiscwidgets! -e s!%PFX%!$(prefix)! <dummy.la >$@
+	sed -e s!%DLL%!$(SOPFX)emiscwidgets-0! -e s!%LIB%!libemiscwidgets! -e s!%LIBDIR%!$(privlibdir)! < $(srcdir)/dummy.la >$@
 	mkdir -p .libs
-	$(DLLTOOL) --output-lib=.libs/libemiscwidgets.dll.a --dllname=libemiscwidgets-0.dll --input-def=libemiscwidgets.def
+	$(DLLTOOL) --output-lib=.libs/libemiscwidgets.dll.a --dllname=$(SOPFX)emiscwidgets-0.dll --input-def=$(srcdir)/libemiscwidgets.def
 
 libetable.la: dummy.la libetable.def
-	sed -e s!%DLL%!libetable-0! -e s!%LIB%!libetable! -e s!%PFX%!$(prefix)! <dummy.la >$@
+	sed -e s!%DLL%!$(SOPFX)etable-0! -e s!%LIB%!libetable! -e s!%LIBDIR%!$(privlibdir)! < $(srcdir)/dummy.la >$@
 	mkdir -p .libs
-	$(DLLTOOL) --output-lib=.libs/libetable.dll.a --dllname=libetable-0.dll --input-def=libetable.def
+	$(DLLTOOL) --output-lib=.libs/libetable.dll.a --dllname=$(SOPFX)etable-0.dll --input-def=$(srcdir)/libetable.def
 
 libetext.la: dummy.la libetext.def
-	sed -e s!%DLL%!libetext-0! -e s!%LIB%!libetext! -e s!%PFX%!$(prefix)! <dummy.la >$@
+	sed -e s!%DLL%!$(SOPFX)etext-0! -e s!%LIB%!libetext! -e s!%LIBDIR%!$(privlibdir)! < $(srcdir)/dummy.la >$@
 	mkdir -p .libs
-	$(DLLTOOL) --output-lib=.libs/libetext.dll.a --dllname=libetext-0.dll --input-def=libetext.def
+	$(DLLTOOL) --output-lib=.libs/libetext.dll.a --dllname=$(SOPFX)etext-0.dll --input-def=$(srcdir)/libetext.def
 
 libevolution-addressbook.la: dummy.la libevolution-addressbook.def
-	sed -e s!%DLL%!libevolution-addressbook! -e s!%LIB%!libevolution-addressbook! -e s!%PFX%!$(prefix)! <dummy.la >$@
+	sed -e s!%DLL%!$(SOPFX)evolution-addressbook! -e s!%LIB%!libevolution-addressbook! -e s!%LIBDIR%!$(componentdir)! < $(srcdir)/dummy.la >$@
 	mkdir -p .libs
-	$(DLLTOOL) --output-lib=.libs/libevolution-addressbook.dll.a --dllname=libevolution-addressbook.dll --input-def=libevolution-addressbook.def
+	$(DLLTOOL) --output-lib=.libs/libevolution-addressbook.dll.a --dllname=$(SOPFX)evolution-addressbook.dll --input-def=$(srcdir)/libevolution-addressbook.def
 
 libevolution-calendar.la: dummy.la libevolution-calendar.def
-	sed -e s!%DLL%!libevolution-calendar! -e s!%LIB%!libevolution-calendar! -e s!%PFX%!$(prefix)! <dummy.la >$@
+	sed -e s!%DLL%!$(SOPFX)evolution-calendar! -e s!%LIB%!libevolution-calendar! -e s!%LIBDIR%!$(componentdir)! < $(srcdir)/dummy.la >$@
 	mkdir -p .libs
-	$(DLLTOOL) --output-lib=.libs/libevolution-calendar.dll.a --dllname=libevolution-calendar.dll --input-def=libevolution-calendar.def
+	$(DLLTOOL) --output-lib=.libs/libevolution-calendar.dll.a --dllname=$(SOPFX)evolution-calendar.dll --input-def=$(srcdir)/libevolution-calendar.def
 
 libevolution-mail.la: dummy.la libevolution-mail.def
-	sed -e s!%DLL%!libevolution-mail! -e s!%LIB%!libevolution-mail! -e s!%PFX%!$(prefix)! <dummy.la >$@
+	sed -e s!%DLL%!$(SOPFX)evolution-mail! -e s!%LIB%!libevolution-mail! -e s!%LIBDIR%!$(componentdir)! < $(srcdir)/dummy.la >$@
 	mkdir -p .libs
-	$(DLLTOOL) --output-lib=.libs/libevolution-mail.dll.a --dllname=libevolution-mail.dll --input-def=libevolution-mail.def
+	$(DLLTOOL) --output-lib=.libs/libevolution-mail.dll.a --dllname=$(SOPFX)evolution-mail.dll --input-def=$(srcdir)/libevolution-mail.def
 
 libfilter.la: dummy.la libfilter.def
-	sed -e s!%DLL%!libfilter-0! -e s!%LIB%!libfilter! -e s!%PFX%!$(prefix)! <dummy.la >$@
+	sed -e s!%DLL%!$(SOPFX)filter-0! -e s!%LIB%!libfilter! -e s!%LIBDIR%!$(privlibdir)! < $(srcdir)/dummy.la >$@
 	mkdir -p .libs
-	$(DLLTOOL) --output-lib=.libs/libfilter.dll.a --dllname=libfilter-0.dll --input-def=libfilter.def
+	$(DLLTOOL) --output-lib=.libs/libfilter.dll.a --dllname=$(SOPFX)filter-0.dll --input-def=$(srcdir)/libfilter.def
 
 CLEANFILES = $(BOOTSTRAP_LIBS)
--- origsrc/evolution-2.26.3/win32/dummy.la	2009-06-26 09:49:49.000000000 -0500
+++ src/evolution-2.26.3/win32/dummy.la	2009-07-27 00:36:37.141922200 -0500
@@ -29,4 +29,4 @@ dlopen=''
 dlpreopen=''
 
 # Directory that this library needs to be installed in:
-libdir='%PFX%/lib'
+libdir='%LIBDIR%'
