--- origsrc/evolution-2.26.2/acinclude.m4	2009-05-03 09:55:30.000000000 -0500
+++ src/evolution-2.26.2/acinclude.m4	2009-06-16 19:33:26.688118300 -0500
@@ -427,6 +427,9 @@ i?86-*-linux*|x86_64-*-linux*|powerpc-*-
 |amd64-*-freebsd*|i?86-*-freebsd*|ia64-*-freebsd*)
     pic_options='-fPIC'
     ;;
+*-*-cygwin*)
+    pic_options='-DDLL_EXPORT'
+    ;;
 i?86-apple-darwin*)
     pic_options='-fno-common'
     ;;
