--- origsrc/evolution-2.26.2/configure.in	2009-05-17 21:45:47.000000000 -0500
+++ src/evolution-2.26.2/configure.in	2009-06-16 19:29:43.817212900 -0500
@@ -144,19 +144,35 @@ PKG_PROG_PKG_CONFIG
 dnl Win32
 AC_MSG_CHECKING([for Win32])
 case "$host" in
+*-cygwin*)
+    platform_win32=yes
+    os_win32=no
+    NO_UNDEFINED='-no-undefined'
+    SOPFX='cyg'
+    SOEXT='.dll'
+    SA_JUNK_PLUGIN=sa-junk-plugin 
+    BF_JUNK_PLUGIN=bogo-junk-plugin
+    DL_LIB=''
+    SOFTOKN3_LIB=''
+    HAL_REQUIREMENT=''
+    ;;
 *-mingw*)
+    platform_win32=yes
     os_win32=yes
     NO_UNDEFINED='-no-undefined'
     SOEXT='.dll'
+    SOPFX='lib'
     SA_JUNK_PLUGIN=''
     BF_JUNK_PLUGIN=''
     DL_LIB=''
     SOFTOKN3_LIB=''
     HAL_REQUIREMENT=''
     ;;
-*)  os_win32=no
+*)  platform_win32=no
+    os_win32=no
     NO_UNDEFINED=''
     SOEXT='.so'
+    SOPFX='lib'
     SA_JUNK_PLUGIN=sa-junk-plugin 
     BF_JUNK_PLUGIN=bogo-junk-plugin
     DL_LIB='-ldl'
@@ -165,9 +181,11 @@ case "$host" in
     ;;
 esac
 AC_MSG_RESULT([$os_win32])
+AM_CONDITIONAL(PLATFORM_WIN32, [test $platform_win32 = yes])
 AM_CONDITIONAL(OS_WIN32, [test $os_win32 = yes])
 AC_SUBST(NO_UNDEFINED)
 AC_SUBST(SOEXT)
+AC_SUBST(SOPFX)
 
 # Check for base dependencies early.
 PKG_CHECK_MODULES(GNOME_PLATFORM,
