--- origsrc/evolution-2.30.1.2/addressbook/conduit/Makefile.am	2010-03-18 09:13:57.000000000 -0500
+++ src/evolution-2.30.1.2/addressbook/conduit/Makefile.am	2010-06-11 05:09:15.693596900 -0500
@@ -23,6 +23,8 @@ e-address.conduit: e-address.conduit.in
 	sed -e 's^\@privconduitdir\@^$(privconduitdir)^g' 		\
 	    -e 's^\@datadir\@^$(datadir)^g' 				\
 	    -e 's^\@BASE_VERSION\@^$(BASE_VERSION)^g' 			\
+	    -e 's^\@SOPFX\@^$(SOPFX)^g'						\
+	    -e 's^\@SOEXT\@^$(SOEXT)^g'						\
 	    $< > $@
 
 conduitdir = $(datadir)/gnome-pilot/conduits/
--- origsrc/evolution-2.30.1.2/addressbook/conduit/e-address.conduit.in	2010-03-18 09:13:57.000000000 -0500
+++ src/evolution-2.30.1.2/addressbook/conduit/e-address.conduit.in	2010-06-11 05:08:03.778483600 -0500
@@ -1,5 +1,5 @@
 <gnome-pilot-conduit version="1.0">
-	<conduit id="e_address_conduit" type="shlib" location="@privconduitdir@/libeaddress_conduit.so"/>
+	<conduit id="e_address_conduit" type="shlib" location="@privconduitdir@/@SOPFX@eaddress_conduit@SOEXT@"/>
 	<name value="EAddress"/>
 	<conduit-attribute name="description" value="Synchronizes Addressbook with Evolution"/>
 	<conduit-attribute name="default-synctype" value="synchronize"/>
--- origsrc/evolution-2.30.1.2/calendar/conduits/calendar/Makefile.am	2010-03-18 09:13:58.000000000 -0500
+++ src/evolution-2.30.1.2/calendar/conduits/calendar/Makefile.am	2010-06-11 05:08:36.796372100 -0500
@@ -26,6 +26,8 @@ e-calendar.conduit: e-calendar.conduit.i
 	sed -e 's^\@privconduitdir\@^$(privconduitdir)^g'		\
 	    -e 's^\@datadir\@^$(datadir)^g' 				\
 	    -e 's^\@BASE_VERSION\@^$(BASE_VERSION)^g' 			\
+	    -e 's^\@SOPFX\@^$(SOPFX)^g'						\
+	    -e 's^\@SOEXT\@^$(SOEXT)^g'						\
 	    $< > $@
 
 conduitdir = $(datadir)/gnome-pilot/conduits/
--- origsrc/evolution-2.30.1.2/calendar/conduits/calendar/e-calendar.conduit.in	2010-03-18 09:13:58.000000000 -0500
+++ src/evolution-2.30.1.2/calendar/conduits/calendar/e-calendar.conduit.in	2010-06-11 05:10:50.449016600 -0500
@@ -1,5 +1,5 @@
 <gnome-pilot-conduit version="1.0">
-	<conduit id="e_calendar_conduit" type="shlib" location="@privconduitdir@/libecalendar_conduit.so"/>
+	<conduit id="e_calendar_conduit" type="shlib" location="@privconduitdir@/@SOPFX@ecalendar_conduit@SOEXT@"/>
 	<name value="ECalendar"/>
 	<conduit-attribute name="description" value="Synchronizes Calendar with Evolution @BASE_VERSION@"/>
 	<conduit-attribute name="default-synctype" value="synchronize"/>
--- origsrc/evolution-2.30.1.2/calendar/conduits/common/Makefile.am	2010-03-18 09:13:58.000000000 -0500
+++ src/evolution-2.30.1.2/calendar/conduits/common/Makefile.am	2010-06-11 05:10:15.767032900 -0500
@@ -14,7 +14,7 @@ ecalendar_common_conduitinclude_HEADERS
 
 libecalendar_common_conduit_la_SOURCES = libecalendar-common-conduit.c
 
-libecalendar_common_conduit_la_LDFLAGS = -module -avoid-version $(NO_UNDEFINED)
+libecalendar_common_conduit_la_LDFLAGS = $(NO_UNDEFINED)
 
 libecalendar_common_conduit_la_LIBADD = 				\
 	$(top_builddir)/e-util/libeutil.la				\
--- origsrc/evolution-2.30.1.2/calendar/conduits/memo/Makefile.am	2010-03-18 09:13:58.000000000 -0500
+++ src/evolution-2.30.1.2/calendar/conduits/memo/Makefile.am	2010-06-11 05:08:53.604333500 -0500
@@ -26,6 +26,8 @@ e-memo.conduit: e-memo.conduit.in
 	sed -e 's^\@privconduitdir\@^$(privconduitdir)^g' 		\
 	    -e 's^\@datadir\@^$(datadir)^g' 				\
 	    -e 's^\@BASE_VERSION\@^$(BASE_VERSION)^g' 			\
+	    -e 's^\@SOPFX\@^$(SOPFX)^g'						\
+	    -e 's^\@SOEXT\@^$(SOEXT)^g'						\
 	    $< > $@
 
 conduitdir = $(datadir)/gnome-pilot/conduits/
--- origsrc/evolution-2.30.1.2/calendar/conduits/memo/e-memo.conduit.in	2010-03-18 09:13:58.000000000 -0500
+++ src/evolution-2.30.1.2/calendar/conduits/memo/e-memo.conduit.in	2010-06-11 05:11:02.744719900 -0500
@@ -1,5 +1,5 @@
 <gnome-pilot-conduit version="1.0">
-	<conduit id="e_memo_conduit" type="shlib" location="@privconduitdir@/libememo_conduit.so"/>
+	<conduit id="e_memo_conduit" type="shlib" location="@privconduitdir@/@SOPFX@ememo_conduit@SOEXT@"/>
 	<name value="EMemos"/>
 	<conduit-attribute name="description" value="Synchronizes Memo List with Evolution @BASE_VERSION@"/>
 	<conduit-attribute name="default-synctype" value="synchronize"/>
--- origsrc/evolution-2.30.1.2/calendar/conduits/todo/Makefile.am	2010-03-18 09:13:58.000000000 -0500
+++ src/evolution-2.30.1.2/calendar/conduits/todo/Makefile.am	2010-06-11 05:09:07.821146600 -0500
@@ -26,6 +26,8 @@ e-todo.conduit: e-todo.conduit.in
 	sed -e 's^\@privconduitdir\@^$(privconduitdir)^g' 		\
 	    -e 's^\@datadir\@^$(datadir)^g' 				\
 	    -e 's^\@BASE_VERSION\@^$(BASE_VERSION)^g' 			\
+	    -e 's^\@SOPFX\@^$(SOPFX)^g'						\
+	    -e 's^\@SOEXT\@^$(SOEXT)^g'						\
 	    $< > $@
 
 conduitdir = $(datadir)/gnome-pilot/conduits/
--- origsrc/evolution-2.30.1.2/calendar/conduits/todo/e-todo.conduit.in	2010-03-18 09:13:58.000000000 -0500
+++ src/evolution-2.30.1.2/calendar/conduits/todo/e-todo.conduit.in	2010-06-11 05:11:14.241377500 -0500
@@ -1,5 +1,5 @@
 <gnome-pilot-conduit version="1.0">
-	<conduit id="e_todo_conduit" type="shlib" location="@privconduitdir@/libetodo_conduit.so"/>
+	<conduit id="e_todo_conduit" type="shlib" location="@privconduitdir@/@SOPFX@etodo_conduit@SOEXT@"/>
 	<name value="EToDo"/>
 	<conduit-attribute name="description" value="Synchronizes ToDo List with Evolution @BASE_VERSION@"/>
 	<conduit-attribute name="default-synctype" value="synchronize"/>
