Already upstreamed.

--- origsrc/evolution-3.2.2/addressbook/gui/widgets/eab-gui-util.c	2011-10-20 05:38:02.000000000 -0500
+++ src/evolution-3.2.2/addressbook/gui/widgets/eab-gui-util.c	2011-11-18 03:36:13.936812900 -0600
@@ -655,8 +655,10 @@ get_locales (void)
 	gchar *dot;
 	gchar **split;
 
+#ifdef LC_ADDRESS
 	locale = g_strdup (setlocale (LC_ADDRESS, NULL));
 	if (!locale)
+#endif
 		return NULL;
 
 	l_locale = g_utf8_strdown (locale, -1);
